marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-ingestion-writer.json

```diff
@@ -1,4 +1,7 @@
 {
   "role-name": "data-hub-ingestion-writer",
-  "description": "Permits reading a load data configuration"
+  "description": "Permits reading a load data configuration",
```

Comment:
should data-hub-ingestion-writer get data-hub-common-writer role just like data-hub-mapping-writer

Reply:
I think that's a good idea.

Reply:
Oh wait I take that back - I think we're just using this as a "data" role, while the hub-central-* roles are "privilege" roles. 

The description does need to be modified though - change "reading" to "writing".

Reply:
I added the data-hub-common-writer role to the data-hub-ingestion-writer. It makes sense to me that the data-hub-ingestion-writer should be enough provide enough privileges/permissions to write an ingestion step outside the context of hub-central.